1.9/networking/service-and-pod-networking.md

```diff
@@ -0,0 +1,422 @@
+---
+post_title: Service and Pod Networking
+menu_order: 0.5
+---
+
+This document describes how to specify networking modes and assign ports in the service definition for a DC/OS Service you are creating.
+
```

Comment:
Do DC/OS users need to know about the networking API changes? it's only relevant if they've modified their native marathon instances AFAIK.